pathoscore.py

```diff
@@ -27,6 +28,48 @@
 
 cmd = "vcfanno -lua {lua} -p {p} {conf} {query_vcf} | bgzip -c > {out_vcf}"
 
+def get_genes(csqs):
+    genes = set()
+    for csq in csqs.split(","):
+        genes.add(csq.split("|", 2)[1])
+    return genes
+
+def clinical_utility(scoredbygene, unscoredbygene, jindices, prefix, goi):
+    genes = set(itertools.chain.from_iterable(map(str,scoredbygene[key].keys()) for key in scoredbygene.keys()))
+    genes = genes.intersection(goi)
+    cu = defaultdict(lambda: dict.fromkeys(genes, 0))
+    tp, tn, fp, fn = 0, 0, 0, 0
+    for method in scoredbygene:
+        for gene in scoredbygene[method]:
+            if gene not in genes:
+                continue
+            scored = len(scoredbygene[method][gene][0])+len(scoredbygene[method][gene][1])
+            unscored = unscoredbygene[method][gene][0]+unscoredbygene[method][gene][1]
+            j = jindices[method]
+            fracvars = scored/float(scored+unscored)
+            for score in scoredbygene[method][gene][1]:
+                if score > j:
```

Comment:
for this block, use:

```Python
if score >= j:
  tp += 1
else:
  fp += 1
```
otherwise, we might missing things that exactly equal the cutoff.